### PR TITLE
Add egress ip workload image

### PR DESCRIPTION
**Pull request by venkataanil**
eipValidator image is used by kube-burner-ocp egress IP workload. nginxEchoIP image is used in e2e-benchmarking egress IP workload to create custom nginx server in external AWS instance.

This PR is required for 
https://github.com/kube-burner/kube-burner-ocp/pull/28
https://github.com/cloud-bulldozer/e2e-benchmarking/pull/699

### CODE REVIEW

File: eipValidator/Containerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/golang:latest
```
Comment (krishvoor) [inline eipValidator/Containerfile:1]: We might run into rate limiting issues with docker.io/ can you instead switch to ubi8 image with Python install, you can refer to this instead https://github.com/cloud-bulldozer/images/blob/main/netpol-scraper/Dockerfile#L1

Comment (krishvoor) [inline eipValidator/Containerfile:1]: Might run into rate limiting issues with docker.io, can you switch to ubi8/9 images something we did with netpol-scraper image

File: eipValidator/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Egress IP validator
```
Comment (krishvoor) [inline eipValidator/README.md:1]: Can you also update this README.md file https://github.com/cloud-bulldozer/images/blob/main/README.md

Comment (venkataanil) [reply eipValidator/README.md:1]: Thanks Harsha. Updated README.md

**Comment by venkataanil** [comment]
I also have built quay.io/vkommadi/eip-checker:latest with the latest Containerfile and tested with kube-burner-ocp

**Review by vishnuchalla — APPROVED** [review]
lgtm. thanks

File: eipValidator/Containerfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    gzip \
+    && microdnf clean all
+
+# Set Go environment variables
+ENV GOLANG_VERSION 1.22.1
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+# Download and install Go
+RUN curl -LO https://golang.org/dl/go${GOLANG_VERSION}.linux-amd64.tar.gz \
```
Comment (krishvoor) [inline eipValidator/Containerfile:15]: Will run into issues with multi-arch containers - can you consider GOARCH to download the right binary.

**Review by krishvoor — APPROVED** [review]
/lgtm

**Comment by krishvoor** [comment]
Tested this locally ( MAC M1 ) works as expected

**Review by jtaleric — APPROVED** [review]
lgtm, i didn't test locally so 🤞🏽

**Pull request merged**